libraries/AP_L1_Control/AP_L1_Control.cpp

```diff
@@ -185,7 +185,7 @@ void AP_L1_Control::_prevent_indecision(float &Nu)
     if (fabsf(Nu) > Nu_limit &&
         fabsf(_last_Nu) > Nu_limit &&
         labs(wrap_180_cd(_target_bearing_cd - get_yaw_sensor())) > 12000 &&
-        Nu * _last_Nu < 0.0f) {
+        Nu * is_negative(_last_Nu)) {
```

Comment:
I think this should be 
`is_negative(Nu *_last_Nu)`
so many traps in this sort of change though